bindings_ffi/src/mls.rs

```diff
@@ -1185,18 +1185,28 @@ impl TryFrom<UserPreferenceUpdate> for FfiPreferenceUpdate {
     }
 }
 
-#[derive(uniffi::Object)]
+#[derive(uniffi::Object, Clone)]
 pub struct FfiConversation {
     inner: MlsGroup<RustXmtpClient>,
 }
 
 #[derive(uniffi::Object)]
-#[allow(unused_variables, dead_code)]
+#[allow(dead_code)]
```

Comment:
if we do this, won't be enough?
```suggestion
#[uniffi::export]
#[allow(dead_code)]
```

Reply:
do we still need the dead_code now that we have `pub fn` to the variables?

Reply:
retracted!

Reply:
It says you cannot export a struct